### DISCUSSION TITLE
Smooth scrolling issue (I've already disable scrolling.smooth)

### DISCUSSION THREAD

**Discussion by franzhuang** [qutebrowser/qutebrowser#6783] [General] [answered]
Hi,I found that when I browser a few pages (such as this: [https://www.techspot.com/article/2349-windows-11-performance/](url)  ) , and scroll down with Ctrl+f，the page scrolls smoothly, but scrolls normally using "Space" key.

I've already disable scrolling.smooth option. Most pages (say 99.9% pages) scroll normally with Ctrl+f, while a few pages act like this. Is there any option I can config ?

**Comment by The-Compiler** [comment]
I don't get any smooth scrolling on that page. Can you reproduce with `--temp-basedir`? Can you show your `:version` information?

**Reply by franzhuang** [reply]
I opened the site again, It's strange that I can't reproduce.
And then I recall that I browsed the page on Pop!OS several hours ago. Now I'm on Fedora34 (dual boot).
The page acts normal on Fedora while scrolls smoothly on Pop! OS.
I can reproduce that using `$ qutebrowser --temp-basedir`.
The version on Pop!OS is as follows:

`qutebrowser v1.10.1
Git commit: 
Backend: QtWebEngine (Chromium 69.0.3497.128)
Qt: 5.12.8

CPython: 3.8.10
PyQt: 5.14.1

sip: 4.19.21
colorama: 0.4.3
pypeg2: 2.15
jinja2: 2.10.1
pygments: 2.10.0
yaml: 5.3.1
cssutils: 1.0.2 $Id$
attr: 19.3.0
PyQt5.QtWebEngineWidgets: yes
PyQt5.QtWebEngine: 5.14.0
PyQt5.QtWebKitWidgets: no
pdf.js: no
sqlite: 3.31.1
QtNetwork SSL: OpenSSL 1.1.1f  31 Mar 2020

Style: QFusionStyle
Platform: Linux-5.13.0-7614-generic-x86_64-with-glibc2.29, 64bit
Linux distribution: Pop!_OS 20.04 LTS (unknown)
Frozen: False
Imported from /usr/lib/python3/dist-packages/qutebrowser
Using Python from /usr/bin/python3
Qt library executable path: /usr/lib/x86_64-linux-gnu/qt5/libexec, data path: /usr/share/qt5
OS Version: `

**Comment by The-Compiler** [comment, marked as answer]
Looks like the website specifically requests smooth scrolling via this in CSS:

```css
html {
    scroll-behavior: smooth;
}
```

see e.g. [scroll-behavior - CSS: Cascading Style Sheets | MDN](https://developer.mozilla.org/en-US/docs/Web/CSS/scroll-behavior).

Note that it says *Note that any other scrolls, such as those performed by the user, are not affected by this property.* - so perhaps older versions of Chromium/QtWebEngine didn't do this correctly, but newer ones do.

**Reply by franzhuang** [reply]
I appreciate all your efforts.
